test/testcafe/spec/DeviceChallengePollViewLoopbackFailfast_spec.js

```diff
@@ -0,0 +1,68 @@
+import { RequestLogger, RequestMock } from 'testcafe';
+import DeviceChallengePollPageObject from '../framework/page-objects/DeviceChallengePollPageObject';
+import IdentityPageObject from '../framework/page-objects/IdentityPageObject';
+import identifyWithDeviceProbingLoopback from '../../../playground/mocks/idp/idx/data/identify-with-device-probing-loopback';
+import loopbackChallengeNotReceived from '../../../playground/mocks/idp/idx/data/identify-with-device-probing-loopback-challenge-not-received';
+
+let failureCount = 0;
+
+const logger = RequestLogger(/introspect|probe|challenge/, { logRequestBody: true, stringifyRequestBody: true });
+
+const mock = RequestMock()
```

Comment:
- do you need to mock `authenticate-polling` endpoint and assert it has been hit?
- feels like we could add extra tests to `DeviceChallengePollView_spec` instead of creating a new test file.

Reply:
New test is added to `DeviceChallengePollViewOktaVerify_spec.js`